Extensions/pokes.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE Pokés **//
-//* VERSION 0.3.0 **//
+//* VERSION 0.3.1 **//
```

Comment:
I increased the patch version because it is not a userfacing feature and a bugfix in so far that it just speed up the extension